tests/testthat/test-content.R

```diff
@@ -168,6 +168,12 @@ without_internet({
       con$content(name = "A name for content"),
       "https://connect.example/__api__/v1/content?name=A%20name%20for%20content&include=tags%2Cowner"
     )
+
+    uuid <- uuid::UUIDgenerate()
+    expect_GET(
+      con$content(guid = uuid, include = "tags"),
+      glue::glue("https://connect.example/__api__/v1/content/{uuid}?include=tags")
```

Comment:
```suggestion
      glue::glue("https://connect.example/__api__/v1/content/{uuid}?include=tags,owner")
```

Reply:
I'm explicitly passing just `"tags"` to `include` here so I think it's correct as written, unless you're suggesting I also change line 174?

Reply:
Oh sorry, you're right, I was thinking about the default value (maybe that's worth a test too), and how the matching in `expect_GET` doesn't require you to specify the exact output, it does substring matching.

Reply:
oh good to know. I'll add a test for the default value and update this one to explicitly pass an argument that doesn't match the default (maybe just swapping the order)